### DISCUSSION TITLE
Evidenzia classe scrutinata

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#633] [Idee e proposte]
Buongiorno,sarebbe possibile evidenziare le classi già scrutinate magari con un colore diverso o con una scritta in modo da sapere quali mancano da scrutinare?Grazie 

P.S.spero nella prossima versione venga implementata la media di ogni singola materia lato genitore/alunno così come per i docenti la casella dove inserire la valutazione e non più trascinare ogni volta il cursore che è pressoché fastidioso e richiede più tempo

**Comment by ghost** [comment]
concordo con Luigi!

**Comment by trinko** [comment]
La segno come richiesta.
